guides/set-current-user.md

```diff
@@ -0,0 +1,97 @@
+[Back to Main README](https://github.com/simplabs/ember-simple-auth/blob/master/README.md)
+
+## How to Set currentUser Data
+
+Although there are various ways to load currentUser data, this is the canonical
+way to do so. Your authentication route will need to identify how it can access
+the data for the current user. In this example, it is sent as the user_id on the
+authentication payload.
+
+### CurrentUser Service
+
+In order for the app to keep the user data, we will store it in its own service:
+
+```js
+// app/services/session-account.js
+import Ember from 'ember';
+
+const { inject: { service }, isEmpty, RSVP } = Ember;
+
+export default Ember.Service.extend({
+  session: service('session'),
+  store: service(),
+
+  account: null,
+
+  loadCurrentUser() {
+    return new RSVP.Promise((resolve, reject) => {
+      const userId = this.get('session.data.authenticated.user_id');
+      if (!isEmpty(userId)) {
+        return this.get('store').find('user', userId).then((user) => {
+          this.set('user', user);
+          resolve();
+        }, reject);
+      } else {
+        resolve();
+      }
+    });
+  }
+});
+```
+
+In this example, we grab the user id and then make an ember-data request to the
+API. After a successful request, it sets the user and is easily access anywhere
+by injecting this service.  Note: If no id is sent, then the user will not be
+set and this implementation will log you out.
```

Comment:
that's not true - the above method will not log the user out but simply not do anything.